### PR TITLE
Wordpress Search function not working

### PR DESCRIPTION
**Pull request by dipskakadiya**
Added code to support global search for page and other custom post types

### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Added search support for page and other custom post types. 

### How to test the changes in this Pull Request:

1. Click on 'search' icon in top navigation menu bar
2. Type in search term with page or Custom CPT like course, lesson etc.
3. See error - no search results returned.

### Proof Screenshots or Video

Screenshot 2020-09-07 at 8 30 38 PM

Screenshot 2020-09-07 at 8 28 16 PM

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Added search support for page and other custom post types

.

### CODE REVIEW

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
@dipskakadiya I don't think we need this changes as I tested in my local with the dev branch and it's working fine without any issues.

FYI: For this we have a settings here https://prnt.sc/uddkuz to enable/disable for the specific post types.

Let me know if I am undestanding wrong here.

**Comment by dipskakadiya** [comment]
@chetansatasiya As We discuss, This issue for the new Setup. With the default setting, it was not working so revise my solution. Please review it.

File: src/bp-search/plugins/search-cpt/index.php [hunk 1]
```diff
@@ -57,11 +57,11 @@ function bp_search_option_cpt_search( $items_to_search ) {
 function bp_search_helpers_cpts( $helpers ) {
 
 	$post_types          = get_post_types( array( 'public' => true ) );
-	$custom_handler_cpts = array( 'post', 'forum', 'topic', 'reply' );
+	$custom_handler_cpts = array( 'post', 'forum', 'topic', 'reply', 'post', 'page' );
```
Comment (chetansatasiya) [inline src/bp-search/plugins/search-cpt/index.php:60]:
```suggestion
	$custom_handler_cpts = array( 'post', 'forum', 'topic', 'reply', 'page' );
```

@dipskakadiya Remove duplicate `post`

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
@dipskakadiya do the requested changes

**Review by chetansatasiya — DISMISSED** [review]
@dipskakadiya Looks good.

@ankitguptaindia Need to test this with the fresh setup and also with the existing site as well.

**Review by mehulkaklotar — APPROVED** [review]
@dipskakadiya this looks good to me as well! 

@ankitguptaindia needs testing here!

**Review by mehulkaklotar — DISMISSED** [review]

**Pull request merged**